src/model/trust.page.model.ts

```diff
@@ -17,13 +17,45 @@ type TrustBeneficialOwnerListItem = {
 };
 
 type TrustWhoIsInvolved = {
-  id: string;
+  trustId: string;
   trustName: string;
   boInTrust: TrustBeneficialOwnerListItem[];
 };
 
+type TrustHistoricalBeneficialOwner = {
+  trustId: string;
+  trustName: string;
+};
+
+interface TrustHistoricalBeneficialOwnerFormBasic {
```

Comment:
Could we call this `TrustHistoricalBeneficialOwnerFormCommon`

Reply:
fixed